cf_units/_udunits2_parser/udunits2Parser.g4

```diff
@@ -1,6 +1,79 @@
 parser grammar udunits2Parser;
-  
+
 options { tokenVocab=udunits2Lexer; }
 
 
-unit_spec: ALL;
+unit_spec:
+    shift_spec? EOF  // Zero or one "shift_spec", followed by the end of the input.
+;
+
+shift_spec:
+    product
+    | product shift number    // Kelvin @ 273.15
+    | product shift timestamp // hours since 2001-12-31 23:59:59.999 +6
```

Comment:
@pelson trivial - add `e.g.` to keep consistent with format of example comments below